cypress.config.ts

```diff
@@ -22,7 +22,6 @@ export default defineConfig({
   reporterOptions: {
     configFile: 'reporter-config.json',
   },
-  videoUploadOnPasses: false,
```

Comment:
Redundant config in newer Cypress, replaced with https://github.com/ministryofjustice/hmpps-template-typescript/blob/486b956683dee06fcaa1332f23233a2f59f048f8/package.json#L26

Reply:
I'm not sure about this - I deliberately added that line a while back, and its purpose is to keep videos of failing tests (in the same way that it keeps screenshots of failing tests)

Having a video of a failing test is more useful than the final screenshot as it shows the journey and/or context of the test, so fixing it is easier

Can we revert this line, and also the corresponding line in `packacge.json` ? (`"int-test": "cypress run",`)

Unless I'm missing something?

Reply:
Ah ha, we need to add `videos: true` to enable video support now. Done 👍